tests/external/get_recordings_test.php

```diff
@@ -267,7 +266,11 @@ public function test_get_recordings_imported() {
         ]); // We need to have a meeting created in order to import recordings.
         $newinstance = instance::get_from_instanceid($newactivity->id);
         $recordings = $instance->get_recordings();
-        end($recordings)->create_imported_recording($newinstance);
+        foreach ($recordings as $rec) {
+            if ($rec->get('name') == 'Recording1') {
```

Comment:
FYI Coding style generally dictates proper, meaningful var names. I would argue that `$rec` should be `$recording`.

Reply:
Agreed. Will fix this.